src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -282,7 +283,10 @@ def validate_encryption(namespace):
     ''' Builds up the encryption object for storage account operations based on the
     list of services passed in. '''
     if namespace.encryption:
-        from azure.mgmt.storage.models import Encryption, EncryptionServices, EncryptionService
+        rt = ResourceType.MGMT_STORAGE_STORAGE_ACCOUNTS
+        Encryption = get_versioned_models(rt, model='Encryption')
+        EncryptionServices = get_versioned_models(rt, model='EncryptionServices')
+        EncryptionService = get_versioned_models(rt, model='EncryptionService')
```

Comment:
Option 1 is fine for now, but I think before we can really move forward with wholesale implementation, we will need something like option 2.